specification/cognitiveservices/data-plane/SpeechServices/stable/v2.0/Speech.json

```diff
@@ -0,0 +1,13520 @@
+            "examples": {
+              "application/json": {
+                "code": "Forbidden",
+                "message": "No permission to access this resource."
+              }
+            }
+          },
+          "415": {
+            "description": "In case the media type of a file part is not supported.",
+            "schema": {
+              "$ref": "#/definitions/ErrorContent"
+            },
+            "headers": {
+              "X-RateLimit-Limit": {
+                "description": "The limit of requests for the resource.",
+                "type": "integer"
+              },
+              "X-RateLimit-Remaining": {
+                "description": "The remaining number of requests until the rate limit is reached.",
+                "type": "integer"
+              },
+              "X-RateLimit-Reset": {
+                "description": "The timestamp when the rate limit will be reset formatted as ISO 8601 combined date and time in UTC.",
+                "type": "string"
+              }
+            },
+            "examples": {
+              "application/json": {
+                "code": "UnsupportedMediaType",
+                "message": "The request entity's media type 'application/xml' is not supported for this resource."
+              }
+            }
+          },
+          "429": {
+            "description": "In case the rate limit has been exceeded.",
+            "schema": {
+              "$ref": "#/definitions/ErrorContent"
+            },
+            "headers": {
+              "X-RateLimit-Limit": {
+                "description": "The limit of requests for the resource.",
+                "type": "integer"
+              },
+              "X-RateLimit-Remaining": {
+                "description": "The remaining number of requests until the rate limit is reached.",
+                "type": "integer"
+              },
+              "X-RateLimit-Reset": {
+                "description": "The timestamp when the rate limit will be reset formatted as ISO 8601 combined date and time in UTC.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The minimum number of seconds to wait before not getting this response anymore.",
+                "type": "integer"
+              }
+            },
+            "examples": {
+              "application/json": {}
+            }
+          }
+        },
+        "security": [
+          {
+            "subscription_key": [],
+            "token": []
+          }
+        ]
+      }
+    },
+    "/api/speechtotext/v2.0/accuracytests/{id}": {
+      "get": {
+        "tags": [
+          "Custom Speech accuracy tests:"
+        ],
+        "summary": "Gets the accuracy test identified by the given ID.",
+        "operationId": "GetAccuracyTest",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
```

Comment:
suggestion: you could abstract the parameter to the global parameter section, since you're referencing from multiple operations.